app-shells/autojump/autojump-22.5.1.ebuild

```diff
@@ -0,0 +1,75 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
```

Comment:
Could you bump to EAPI version 6 whilst at it?

Reply:
Done. I've needed to add `eapply_user`.